src/bench/bloom.cpp

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2016 The Bitcoin Core developers
```

Comment:
Both date and Copyright owner seem not correct.

Reply:
thanks, fixed